src/app.ts

```diff
@@ -208,6 +208,12 @@ async function app() {
     ) ?? [];
   const fhirPackageId = determineCorePackageId(config.config.fhirVersion[0]);
   allDependencies.push(`${fhirPackageId}@${config.config.fhirVersion[0]}`);
+  await utils.loadAutomaticDependencies(
+    config.config.fhirVersion[0],
+    config.config.dependencies ?? [],
+    // @ts-ignore TODO: this can be removed once SUSHI changes the type signature for this function to use FPL's FHIRDefinitions type
+    defs
+  );
   const dependencyDefs = loadExternalDependencies(defs, allDependencies);
```

Comment:
I have a question about the location of this new function. Do you know if it is possible/if it is useful to include the `loadAutomaticDependencies` within the `loadExternalDependencies` function? To me, it seems like it logically makes sense there since they are part of the set of external dependencies that are loaded, and it seems like SUSHI loads the automatic dependencies from within its `loadExternalDependencies` function. That would also allow us to add a test to ensure the automatic dependencies are actually included, since the function wouldn't just be within `app.ts`.

That said, I'm curious what your thoughts are on this.